sphinx/source/ledger.rst

```diff
@@ -21,48 +21,48 @@ Note that even if a transaction only affects a private ``Store::Map``, unencrypt
 Ledger replication
 ------------------
 
-The replication process currently uses Raft as the consensus algorithm. As such, it relies on authenticated Append Entries (AE) headers sent from the leader to followers and which specify the start and end index of the encrypted deltas payload. When an AE header is emitted from a node's enclave for replication, the corresponding encrypted deltas are read from the ledger and appended to the AE header.
+The replication process currently uses Raft as the consensus algorithm. As such, it relies on authenticated Append Entries (AE) headers sent from the primary to backups and which specify the start and end index of the encrypted deltas payload. When an AE header is emitted from a node's enclave for replication, the corresponding encrypted deltas are read from the ledger and appended to the AE header.
 
-The following diagram describes how deltas committed by the leader are written to the ledger and how they are replicated to one follower. Note that the full replication process and acknowledgment from the follower is not detailed here.
+The following diagram describes how deltas committed by the primary are written to the ledger and how they are replicated to one backup. Note that the full replication process and acknowledgment from the backup is not detailed here.
```

Comment:
I think we should add a note somewhere in the docs, probably around here, explaining why we're talking about Raft but not using Raft terminology. ie  - we intend to support multiple consensus algorithms, so this documentation and much of the code uses abstract, consensus-agnostic terms "primary" and "backup". These correspond directly with the "leader" and "follower" roles in Raft literature.

Reply:
On second thought, perhaps we can just describe this in the Glossary.

Reply:
Actually it would make sense to replace raft with consensus (or consensus protocol) wherever it is used generically in the documentation, and make a note that we are currently using raft for consensus and in that case primary maps to leader, backup maps to follower, and seqno->index, view->term

Reply:
True, but in this case I think the description and diagram are still Raft-specific, so I think we should use Raft terminology or explain why we're not.